crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -156,6 +116,110 @@ pub async fn create(
     })
 }
 
+pub async fn import_from_keystore(
+    provider: &JsonRpcClient<HttpTransport>,
+    keystore_path: Utf8PathBuf,
+    account_path: Utf8PathBuf,
+) -> Result<(serde_json::Value, u64)> {
+    let account_info: serde_json::Value = serde_json::from_str(&fs::read_to_string(account_path)?)?;
+    let deployment = account_info
+        .get("deployment")
+        .ok_or_else(|| anyhow!("No deployment field in account JSON file"))?;
+
+    let deployed = match deployment.get("status").and_then(serde_json::Value::as_str) {
+        Some("deployed") => true,
+        Some("undeployed") => false,
+        _ => bail!("Unknown deployment status value"),
+    };
+
+    let salt: Option<FieldElement> = {
+        let salt = deployment.get("salt").and_then(serde_json::Value::as_str);
+        match (salt, deployed) {
+            (Some(salt), _) => Some(parse_number(salt)?),
+            (None, false) => bail!("No salt field in account JSON file"),
```

Comment:
I think a clear error is better here instead of implicit behaviour.
I don't think this would happen, if someone uses starkli to generate the account json file anyway.